packages/connectivity/src/scp-cf/destination/destination-service-types.ts

```diff
@@ -190,7 +190,14 @@ export interface DestinationCertificate {
 /**
  * Options to use while fetching destinations. Encompasses both [[DestinationCachingOptions]] and [[ResilienceOptions]] interfaces.
  */
-export type DestinationRetrievalOptions = CachingOptions & ResilienceOptions;
+export type DestinationRetrievalOptions = CachingOptions &
+  ResilienceOptions & {
+    /**
+     * The isolation strategy used for caching destinations. For the available options, see [[IsolationStrategy]].
+     * By default, IsolationStrategy.Tenant is set.
+     */
+    isolationStrategy?: IsolationStrategy;
+  };
```

Comment:
It seems the line 203 is misleading, which should be `Destination` instead of `DestinationFetchOptions`.